modules/light-clients/09-localhost/client_state.go

```diff
@@ -153,8 +169,10 @@ func (cs ClientState) UpdateStateOnMisbehaviour(_ sdk.Context, _ codec.BinaryCod
 func (cs ClientState) UpdateState(ctx sdk.Context, cdc codec.BinaryCodec, clientStore sdk.KVStore, clientMsg exported.ClientMessage) []exported.Height {
 	height := clienttypes.GetSelfHeight(ctx)
 
-	clientState := NewClientState(ctx.ChainID(), height)
-	clientStore.Set([]byte(host.KeyClientState), clienttypes.MustMarshalClientState(cdc, clientState))
+	cs.ChainId = ctx.ChainID()
```

Comment:
Maybe should be a separate PR. But do we need the chainID if the function got removed above